.github/workflows/formula.yml

```diff
@@ -88,28 +88,28 @@ jobs:
           RESOURCES_FILE_PATH=$BUILD_DIR/resources.txt
           poet -r ${{ github.event.inputs.dbt-package }} | .github/process-python-resources.py ${{ github.event.inputs.dbt-package }} > "$RESOURCES_FILE_PATH"
           cat "$RESOURCES_FILE_PATH"
-          echo ::set-output name=formula-resources-file::"$RESOURCES_FILE_PATH"
+          echo "formula-resources-file=${RESOURCES_FILE_PATH}" >> $GITHUB_OUTPUT
 
       - name: Generate formula file name
         id: gen-filename
         run: |
           if [ ${{ steps.semver.outputs.is-pre-release }} == 1 ]
           then
             echo "This is a prelease"
-            echo ::set-output name=filename-with-version::"Formula/${{ github.event.inputs.dbt-package }}@${{ steps.semver.outputs.base-version }}-${{ steps.semver.outputs.pre-release }}.rb"
+            echo filename-with-version="Formula/${{ github.event.inputs.dbt-package }}@${{ steps.semver.outputs.base-version }}-${{ steps.semver.outputs.pre-release }}.rb" >> $GITHUB_OUTPUT
           else
             echo "This is a final release"
-            echo ::set-output name=filename-with-version::"Formula/${{ github.event.inputs.dbt-package }}@${{ steps.semver.outputs.version }}.rb"
+            echo filename-with-version="Formula/${{ github.event.inputs.dbt-package }}@${{ steps.semver.outputs.version }}.rb" >> $GITHUB_OUTPUT
           fi
-          echo ::set-output name=filename::"Formula/${{ github.event.inputs.dbt-package }}.rb"
+          echo filename="Formula/${{ github.event.inputs.dbt-package }}.rb" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
          echo "filename-with-version=${FILENAME_WITH_VERSION}" >> $GITHUB_OUTPUT
          FILENAME="Formula/${{ github.event.inputs.dbt-package }}.rb"
          echo "filename=${FILENAME}" >> $GITHUB_OUTPUT
```

Reply:
The above three changes all go together and make the assignment a little more clear.  What you had wasn't wrong but this makes the workflow a little easier to read.